### ISSUE TITLE
Add integration test for wipe-history

### ISSUE THREAD

**Issue by mkenigs** [flox/flox#125]
It would be good to add an integration test for wipe-history that exercises `nix store gc`, but in order to do so it seems we'd need to better support `NIX_STORE_DIR` (e.g. currently a `flox install` fails when that variable is set).

We could cleanup something to this effect:
```
#!/usr/bin/env bash

flox destroy -e demo -f
flox install -e demo parallel
flox remove -e demo parallel
flox install -e demo bat
flox remove -e demo bat
flox install -e demo bat
flox remove -e demo bat
flox install -e demo bat
flox remove -e demo bat
flox install -e demo bat
flox remove -e demo bat
flox install -e demo bat
flox remove -e demo bat
branch="aarch64-darwin.demo"
handle="mkenigs"
rm -rf "/tmp/$handle"
git -C /tmp clone ~/.cache/flox/meta/mkenigs
git -C "/tmp/$handle" checkout "$branch"
flox git show "$branch":metadata.json | jq '.generations."1".lastActive |= 0' > "/tmp/$handle/metadata.json"
git -C "/tmp/$handle" commit -m "Set lastActive to 0" metadata.json
git -C "/tmp/$handle" push
find ~/.local/share/flox/environments/$handle -name "*demo*"

echo "Before"
find ~/.local/share/flox/environments/$handle -name "*demo*" | wc -l
parallel_bin=$(readlink $(readlink /Users/matthew/.local/share/flox/environments/mkenigs/aarch64-darwin.demo-1-link/bin))
echo "$parallel_bin"

./result/bin/flox wipe-history -e demo

echo "After"
find ~/.local/share/flox/environments/$handle -name "*demo*" | wc -l
ls "$parallel_bin"
```

**Comment by stahnma** [comment]
@mkenigs Does this attach to an issue?

**Comment by mkenigs** [comment]
Not sure what you mean - this is a new issue

**Comment by limeytexan** [comment]
Possibly blocked on issue to do targeted GC of the nix store.
